app/controllers/observation_quality_experiments_controller.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+class ObservationAccuracyExperimentsController < ApplicationController
+  layout "bootstrap"
+
+  def get_more_validators
+    experiment_id = params[:id]
+    @experiment = ObservationAccuracyExperiment.find( experiment_id )
+    @more_validators = @experiment.get_validator_names( limit: nil, offset: 20 )
+    respond_to do | format |
+      format.html { render partial: "additional_validators", locals: { validators: @more_validators } }
+    end
+  end
+
+  def show
+    @experiment = ObservationAccuracyExperiment.find( params[:id] )
```

Comment:
This method could leverage the same before_action if you define one based on my above comment